doc: quick_start: edits from testing (#17357)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_lbs/README.rst

Testing
=======

> After programming the sample to your dongle or development kit, the sample starts the advertising loop and one of the LEDs starts blinking to indicate this (see `User interface`_ for details).

Reviewer: ```suggestion
After programming the sample to your dongle or development kit, one of the LEDs starts blinking, indicating the advertising loop is active (see `User interface`_ for details).
```
